manifests/p/Posit/Quarto/1.3.340/Posit.Quarto.locale.zh-CN.yaml

```diff
@@ -0,0 +1,48 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
+
+PackageIdentifier: Posit.Quarto
+PackageVersion: 1.3.340
+PackageLocale: zh-CN
+Publisher: Rstudio, PBC

```

Comment:
@dpprdan Just wondering why not change the Publisher ultimately ? 
I think you did that for the other tool right ? https://github.com/microsoft/winget-pkgs/pull/100319

Reply:
@cderv You're right, Publisher should be Posit now. I see you opened https://github.com/microsoft/winget-pkgs/pull/105250. 👍🏻

Reply:
@cderv Now I remember why I didn't change the Publisher to Posit: 
"Rstudio, PBC" (sic!) is still showing up as the Publisher in the ARP dialog. (For Posit.RStudio it is indeed Posit Software)

![grafik](https://user-images.githubusercontent.com/1423562/235933358-8dc6c538-2fc3-4e89-94c7-383d31c4a493.png)

[SandboxTest.ps1](https://github.com/microsoft/winget-pkgs/blob/master/FAQ.md#how-do-i-get-the-appsandfeaturesentries-for-an-installer) reports accordingly:

```
--> Installing WinGet
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Admin setting enabled.Admin setting enabled.

--> Installing the Manifest 1.3.340

Found Quarto [Posit.Quarto] Version 1.3.340
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/quarto-dev/quarto-cli/releases/download/v1.3.340/quarto-1.3.340-win.msi
  ██████████████████████████████  77.8 MB / 77.8 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName DisplayVersion Publisher    ProductCode
----------- -------------- ---------    -----------
Quarto      1.3.340        Rstudio, PBC {84C4DFFB-EAE4-4023-BF77-E56014E4D240}

```

My guess would be that it needs to be changed here:
https://github.com/quarto-dev/quarto-cli/blob/4a4e9f1226df1f6f994ac084b226150966bc7aa5/package/src/windows/quarto.wxs#L4

Vaguely related, I also found [this](https://github.com/quarto-dev/quarto-cli/blob/4a4e9f1226df1f6f994ac084b226150966bc7aa5/package/src/linux/installer.ts#L82) and [that](https://github.com/quarto-dev/quarto-cli/blob/4a4e9f1226df1f6f994ac084b226150966bc7aa5/package/src/linux/installer.ts#L108).